src/app/(Authentication)/Login/Password/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { router, useLocalSearchParams } from 'expo-router';
+import React, { useState } from 'react';
+import { Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import { passwordExists, signInUser } from '../../../../supabase/queries/auth';
+
+export default function LoginScreen() {
+  const [password, setPassword] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+  const email = useLocalSearchParams() as unknown as string;
```

Comment:
This is actually a little funky and assigns `email` to `{"email": "email@email.com"}`. For now, I fixed this by doing `const { email } = useLocalSearchParams<{ email: string }>();
  const userEmail = email as string;` but I will do more research and get back to you if there is a better fix!